src/content/en/agencies/partials/web-directory.html

```diff
@@ -0,0 +1,71 @@
+{% framebox class="directory-framebox" height="auto" enable_widgets="true" %}
+  <link rel="stylesheet" type="text/css" href="/web/agencies/css/directory.css">
+  <div class="filters">
+    <div class="filters__search">
+      <input class="filters__search__field" placeholder="Search for web agencies" type="text" name="agency-search" value="" autocomplete="off" aria-label="Search box">
+      <div class="devsite-search-image material-icons"></div>
+    </div>
+    <div class="filters__dropdowns"></div>
+  </div>
+  <div id="map"></div>
+  <div class="table__bar">
+    <div class="table__logo"></div>
+    <div class="table__info table__heading mobile-is-hidden" data-sort-key="agency_name" data-sort-by="alphabetically">
+      <span class="material-icons">arrow_right_alt</span>Agency name
+    </div>
+    <div class="table__lighthouse-score table__heading mobile-is-hidden" data-sort-key="max_lighthouse_score" data-sort-by="numerically" data-sort-ascending="false">
+      <span class="material-icons">arrow_right_alt</span>Highest Lighthouse Score
+    </div>
+  </div>
+  <div class="directory-framebox__wrapper"></div>
+  <script src="/web/agencies/js/directory.js" type="text/javascript"></script>
+  <script>
+    // Markup for each card below
+    var markup = "<div class='agency__logo mobile-is-hidden'><% if (this.image_url.length) { %><img src='<% this.image_url %>' alt='<% this.agency_name %> logo'/><% } %></div>";
+    markup += "<div class='agency__info'><p class='agency__name'><% this.agency_name %></p>";
+    markup += "<div class='agency__urls'><h4>" +
```

Comment:
Hmm. This looks pretty tough to maintain. We're also introducing a templating syntax into the site :\

Reply:
Ah I didn't know there was going to be a templating engine in the future, we were trying to make it more flexible for different types of markup, but I know it's not pretty :)  I'll update this to be created purely from JS then.

Reply:
Sorry, I mean that you're introducing a new templating solution into the site.

Reply:
Gotcha, I removed it regardless.  We were trying to think of a solution where it wasn't something like a switch statement that depends on 'type' of markup, since the markup for these cards will not be consistent throughout all directories.